yt/utilities/lib/grid_traversal.pxd

```diff
@@ -29,11 +29,26 @@ ctypedef void sampler_function(
                 int index[3],
                 void *data) nogil
 
+#-----------------------------------------------------------------------------
+# walk_volume(VolumeContainer *vc,  np.float64_t v_pos[3], np.float64_t v_dir[3], sampler_function *sample,
+#             void *data, np.float64_t *return_t = NULL, np.float64_t max_t = 1.0)
+#      vc        VolumeContainer*  : Pointer to the volume container to be traversed.
+#      v_pos     np.float64_t[3]   : The x,y,z coordinates of the ray's origin.
+#      v_dir     np.float64_t[3]   : The x,y,z coordinates of the ray's direction.
+#      sample    sampler_function* : Pointer to the sample function to be used.
+#      return_t  np.float64_t*     : # TODO: Unsure of behavior. Defaulted to NULL.
+#      max_t     np.float64_t      : # TODO: Unsure of behavior. Defaulted to 1.0.
```

Comment:
@matthewturk 
1. Is ```max_t``` the maximum time allowed during a voxel traversal?
2. Why is it defaulted to 1.0?

Reply:
So in the terminology we're using here, `t` isn't "time" but rather the "parameter."  The setup is that for each vector, we have a starting point (`v_0`) and a terminating point (`v_1`), and it's scaled such that `v_0 + v * t` is equal to `v_0` at t = 0 and `v_1` at t = 1.  It's set to 1.0 by default because we allow it to go to the end of the vector.